packages/flutter/lib/src/material/stepper.dart

```diff
@@ -111,6 +113,12 @@ class Step {
 
   /// Whether or not the step is active. The flag only influences styling.
   final bool isActive;
+
+  /// title and subtitle below the StepIcon, default value false
+  final bool isStepperTypeHorizontalBottom;
+
+  /// line flow StepIcon's midY, default value false
+  final bool isStepperTypeHorizontalBottomLineFollowIconMidY;
```

Comment:
Following the enum idea above, a more clear name for this attribute might be something like `horizontalLinePosition`, with values of `enum HorizontalLinePosition { center, top }`.